dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -52,44 +52,24 @@
     $wgMWLoggerDefaultSpi = [
         'class' => \WBStack\Logging\CustomSpi::class,
         'args' => [[
-            'ignoreLevels' => [
-                'debug',
-                'info',
-            ],
-            'ignoreAllInGroup' => [
-                'DBPerformance',
-                'objectcache',// ideally want to show objectcache errors, but not warnings
-            ],
-            'logAllInGroup' => [
-                'WBSTACK',
```

Comment:
I believe previously all debug and info level things in this group were logged where as now they will not be